### DISCUSSION TITLE
Prefetching does not work when an element contains other elements

### DISCUSSION THREAD

**Discussion by erikwestlund** [livewire/livewire#1249] [4. General]
Prefetching should send one request to an action, then see if an item gets clicked, and if it does, swap out the component with the state from after the action has run.

This works fine:

```
<span wire:click.prefetch="show">
  Test
</span>
```

This, however, does not:

```
<span wire:click.prefetch="show">
  <svg ....>
</span>
```

What happens is, if you hover, it sends a network request. But then when you click, it sends another one -- essentially rendering the prefetch useless.

For example, this:

```
<span wire:click.prefetch="show">
  <span>Test</span>
</span>
```

Won't even let you run the show action a second time.

I'm not sure if this is expected behavior, but it does make it so I can only prefetch on basic test span elements and cannot use SVG icons for buttons.  Any insight is welcome.

**Comment by calebporzio** [comment]
This should work. I think you should submit an issue and hopefully a PR if you can dig deeper
